website/docusaurus.config.js

```diff
@@ -15,6 +15,14 @@ const config = {
     colorMode: {
       respectPrefersColorScheme: true,
     },
+    announcementBar: {
+      id: 'support_us',
+      content:
+        'Android app unavailable in the Google Play Store store. Please check out installing manually by side-loading or using the Amazon Apps method  <a href="/docs/software/android/android-installation" href="#">here</a>',
```

Comment:
I'd change the "Please check out installing manually..." Bit to "Please install from GitHub or Amazon Store. And have the text be the link.